src/config.py

```diff
@@ -0,0 +1,87 @@
+import json
+import logging
+import os.path
+import re
+import secrets
+import string
+
+import jsonschema
+from src.ssm import get_parameter
+
+LOGGER = logging.getLogger("root")
+
+
+def generate_password():
+    alphabet = string.ascii_letters + string.digits
+    password = "".join(secrets.choice(alphabet) for i in range(20))
+    return password
+
+
+def is_valid(data):
+    schema = load_config("schema/config.json")
+    try:
+        jsonschema.validate(data, schema)
+    except jsonschema.exceptions.ValidationError as err:
+        print(err)
+        return False
+    return True
+
+
+def load_config(filename):
+    if not is_config_exists(filename):
+        LOGGER.error("Provided configuration file is not available")
+        exit(1)
```

Comment:
We should not have program exit nested deep into the code.  If `is_config_exists` returns false, `load_config` could simply return `None` and `is_valid` can return false if there is no schema.